jsweb/exercises/dom_detective.html

```diff
@@ -2,67 +2,63 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <title>Project: DOM Detective</title>
+    <title>Exercise: DOM Access</title>
```

Comment:
Doesn't matter; I'd rather keep the way it is. I just changed the file name per your instructions.